bioconda_utils/docker_utils.py

```diff
@@ -198,6 +198,7 @@ def __init__(
         use_host_conda_bld=False,
         pkg_dir=None,
         keep_image=False,
+        image_build_dir=None,
```

Comment:
I had to add this to be able to test the repository's `Dockerfile`.
Since `Dockerfile` does `COPY . /tmp/repo`, those to-be-copied files have to exist in the image build directory. Previously that directory couldn't be prepopulated with any other files and only contained `Dockerfile` and `requirements.txt`.

Reply:
👍